src/main/java/nl/knaw/dans/bridge/plugin/lib/common/IBagitComposer.java

```diff
@@ -0,0 +1,14 @@
+package nl.knaw.dans.bridge.plugin.lib.common;
+
+import nl.knaw.dans.bridge.plugin.lib.exception.BridgeException;
+
+import java.io.File;
+import java.net.URL;
+import java.util.Map;
+
+/*
+ * @author: Eko Indarto
+ */
+public interface IBagitComposer {
```

Comment:
We use Bagit, but it could be something else, in a more general description I would say its the dataset deposit data (and metadata), in one file possibly a zip.

Reply:
Done!
👍